include/ola/acn/ACNVectors.h

```diff
@@ -87,6 +90,15 @@ enum E133ControllerVector {
   VECTOR_CONTROLLER_EXPECT_MASTER = 5,  /**< Expect master message */
 };
 
+/**
+ * @brief Vectors used at the E1.33 LLRP layer.
+ */
+enum LLRPVector {
+  VECTOR_LLRP_PROBE_REQUEST = 1,  /**< LLRP Probe Request */
+  VECTOR_LLRP_PROBE_REPLY = 1,  /**< LLRP Probe Reply */
```

Comment:
According to the Internet it's not an error, but an option.

I wondered if it was multiple vectors at this level, but it seems it's actually just a typo which I fixed here:
https://github.com/OpenLightingProject/ola/commit/956c0fb0eaedb81a303d22cbcbe60239eebe0c1a

I'm tempted to ignore this for now given the tests are passing (but good spot!). Feels like it might get into a slippery slope of if we pull in that commit, the tests might not pass and there may be other commits where I've not been as separate about which files are changed in which commit and we might risk the whole of LLRP being pulled in. I've already had a few bits like this just getting this far.

Reply:
I'd suggest you cherry-pick that commit to this PR. But if the plan is to merge the rest of E1.33 soon-ish, we can just merge this one as-os

Reply:
Leaving for a future PR which will pull them in.